tests/pytests/common.py

```diff
@@ -84,7 +84,12 @@ def getConnectionByEnv(env):
 def waitForIndex(env, idx = 'idx'):
     waitForRdbSaveToFinish(env)
     while True:
-        res = env.cmd('ft.info', idx)
+        try:
+            res = env.cmd('ft.info', idx)
+        except redis_exceptions.ResponseError as e:
+            if 'redis is busy' in str(e).lower():
+                continue
+            raise e
```

Comment:
Consider handling the exception this way, so we don't skip the `sleep` call at the end of the loop
```suggestion
        except redis_exceptions.ResponseError as e:
            if 'redis is busy' not in str(e).lower():
                raise e
```